src/ol/webgl/Buffer.js

```diff
@@ -35,7 +35,6 @@ export const BufferUsage = {
  * Note:
  * See the documentation of [WebGLRenderingContext.bufferData](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/bufferData)
  * for more info on buffer usage.
- * @api
```

Comment:
Did your pull request change anything in WebGLArrayBuffer? If not you could leave this `@api` annotation here, and we can remove it for the next major release.

Reply:
yep, that could work